src/st/st0_psp/3101C.c

```diff
@@ -1,16 +1,352 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
-#include "common.h"
+#include "../st0/st0.h"
+#include "../st0/dialogue2.h"
 
-INCLUDE_ASM("st/st0_psp/psp/st0_psp/3101C", func_pspeu_09242A20);
+typedef struct {
+    Primitive* prim;
+    float y0;
```

Comment:
This is quite peculiar, don't think there are many places outside DRA PSP using floats, but here is one. It doesn't seem to have much purpose as far as I can tell, just gets set here from a function below.